catalog/core/catalog-core-standardframework/src/test/groovy/ddf/catalog/impl/operations/ResourceOperationsSpec.groovy

```diff
@@ -34,7 +34,7 @@ import spock.lang.Unroll
 
 import java.util.concurrent.ExecutorService
 
-class ResourceOperationsTest extends Specification {
+class ResourceOperationsSpec extends Specification {
```

Comment:
✏️ There's a pretty good chance that this change has bumped up test coverage numbers in this bundle. Probably worth checking.